Mathlib/Algebra/Order/Floor.lean

```diff
@@ -136,6 +129,21 @@ theorem le_floor (h : (n : α) ≤ a) : n ≤ ⌊a⌋₊ :=
   (le_floor_iff <| n.cast_nonneg.trans h).2 h
 #align nat.le_floor Nat.le_floor
 
+theorem gc_ceil_coe : GaloisConnection (ceil : α → ℕ) (↑) :=
+  FloorSemiring.gc_ceil
+#align nat.gc_ceil_coe Nat.gc_ceil_coe
+
+@[simp]
+theorem ceil_le : ⌈a⌉₊ ≤ n ↔ a ≤ n :=
+  gc_ceil_coe _ _
+#align nat.ceil_le Nat.ceil_le
```

Comment:
`#align` has been removed meanwhile.

Reply:
I do not have experience with merging, I hope the conflicts have been resolved.